civis/cli/_cli_commands.py

```diff
@@ -85,6 +86,71 @@ def notebooks_download_cmd(notebook_id, path):
             f.write(lines)
 
 
+@click.command('new')
+@click.argument('language', type=click.Choice(['python3', 'python2', 'r']),
+                default='python3')
+@click.option('--mem', type=int, default=None,
+              help='Memory allocated for this notebook in MiB.')
+@click.option('--cpu', type=int, default=None,
+              help='CPU available for this notebook in 1/1024 of a core.')
```

Comment:
I conferred with Saranga and for K8S which powers notebooks, the units are 1/1000 of a core. Container scripts are 1/1024.

Reply:
Ugh. Okay, changed back.